exercises/chapter6/test/Main.purs

```diff
@@ -208,7 +208,7 @@ Note to reader: Delete this line to expand comment block -}
             Assert.equal (act m1 (act m2 a))
               $ act (m1 <> m2) a
           test "Multiply Array Int concrete" do
```

Comment:
Don't know if "concrete" is still the best name here, but can't think of anything better at the moment.

Reply:
How about "Acts on array elements"?

Reply:
This changes the schema of test-naming though. Maybe do as part of #138.

Reply:
> How about "Acts on array elements"?

Sounds good to me